web/src/copy_and_paste.ts

```diff
@@ -542,7 +546,10 @@ export function paste_handler_converter(paste_html: string): string {
             );
         },
 
-        replacement(_content, node, options) {
+        replacement(content, node, options) {
+            if (compose_state.message_content().at(-1) === "`") {
+                return content;
+            }
```

Comment:
This probably belongs in the `!code.includes("\n")` block? Not sure, but certainly for a multi-line code block, we would want at least a leading newline for a good experience.

Reply:
That makes sense, we can start with just doing single line block for now.